content/guides/deploying-a-gitlab-cluster-on-gke/index.md

```diff
@@ -0,0 +1,541 @@
+variable "db_name_prefix" {
+  description = "The name prefix for the database instance. Will be appended with a random string. Use lowercase letters, numbers, and hyphens. Start with a letter."
+}
+
+variable "postgres_version" {
+  description = "The engine version of the database, e.g. `POSTGRES_9_6`. See https://cloud.google.com/sql/docs/db-versions for supported versions."
+  default     = "POSTGRES_9_6"
+}
+
+variable "db_machine_type" {
+  description = "The machine type to use, see https://cloud.google.com/sql/pricing for more details"
+  default     = "db-n1-standard-1"
+}
+
+variable "db_name" {
+  description = "Name for the db"
+  default     = "gitlab"
+}
+
+variable "master_ipv4_cidr_block" {
+  description = "The IP range in CIDR notation (size must be /28) to use for the hosted master network. This range will be used for assigning internal IP addresses to the master or set of masters, as well as the ILB VIP. This range must not overlap with any other ranges in use within the cluster's network."
+  default     = "10.5.0.0/28"
+}
+
+# For the example, we recommend a /16 network for the VPC. Note that when changing the size of the network,
+# you will have to adjust the 'cidr_subnetwork_width_delta' in the 'vpc_network' -module accordingly.
+variable "vpc_cidr_block" {
+  description = "The IP address range of the VPC in CIDR notation. A prefix of /16 is recommended. Do not use a prefix higher than /27."
+  default     = "10.3.0.0/16"
+}
+
+# For the example, we recommend a /16 network for the secondary range. Note that when changing the size of the network,
+# you will have to adjust the 'cidr_subnetwork_width_delta' in the 'vpc_network' -module accordingly.
+variable "vpc_secondary_cidr_block" {
+  description = "The IP address range of the VPC's secondary address range in CIDR notation. A prefix of /16 is recommended. Do not use a prefix higher than /27."
+  default     = "10.4.0.0/16"
+}
+
+```
+
+**Note:** Be sure to fill in any required variables that don't have a default value.
+
+Now we can use Terraform to create the resources:
+
+1. Run `terraform init`.
+2. Run `terraform plan`.
+3. If the plan looks good, run `terraform apply`.
+
+Terraform will begin to create the GCP resources. This process can take up to XX minutes.
+
+## Installing GitLab
+
+We install GitLab by following the recommended way using a Helm chart. The chart contains all of the required components
+to run GitLab and can easily scale to large deployments.
+
+More information about the chart is explain on the GitLab cloud native Helm chart page. [GitLab cloud native Helm Chart | GitLab](https://docs.gitlab.com/charts/)
+
+**Note:** Some GitLab features are currently not available when using the Helm chart:
+
+- GitLab Pages
+- GitLab Geo
+- No in-cluster HA database
+- Smartcard authentication
+
+We can check the latest versions using the following command:
+
+```bash
+$ helm search -l gitlab/gitlab | head -n4
```

Comment:
This doesn't actually install the chart. Is that intentional?

Reply:
@yorinasub17 yep purely for listing available versions, but I need to add an install section.